cdap-docs/admin-manual/source/operations/tx-maintenance.rst

```diff
@@ -0,0 +1,19 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+===============================
+Transaction Service Maintenance
+===============================
+
+.. highlight:: console
+
+The Transaction Service keeps track of all invalid transactions to exclude their writes from all future reads. 
+Over time this invalid list can grow and lead to performance degradation. To avoid that, you can prune the invalid 
```

Comment:
```
Over time, this *invalid* list can grow and lead to performance degradation. 
To avoid a performance hit, you can prune the *invalid * list after the data of the invalid transactions 
has been removed |---| which happens during major HBase compactions of the 
transactional tables.
```